cypress/e2e/buy-shirt.cy.ts

```diff
@@ -24,13 +24,13 @@ describe("Buy a t-shirt", () => {
   it("then should be bought a t-shirt", () => {
     menuContentPage.visitMenuContentPage();
     menuContentPage.goToTShirtMenu();
-    productListPage.addcart();
-    productListPage.pcheckout();
-    shopingCartPage.pcheckout();
+    productListPage.addTocart();
+    productListPage.ProceedToCheckout();
+    shopingCartPage.ProceedTocheckout();
```

Comment:
"A function with a name starting with an uppercase letter should only be used as a constructor"
Change the name to starting with a lower letter `proceedToCheckout();`